### PR TITLE
Update to MySQL EF Core Open Source Providers

### PR DESCRIPTION
**Pull request by adebisi-fa**
* Initial support for EF Core by the Official MySQL Provider
* Full Support by Pomelo Foundation (more complete)

EntityFramework Issue reference #5659.

### CODE REVIEW

File: docs/providers/mysql/index.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 MySQL (Coming Soon)
 ===================
 
-We are working with the MySQL team to make a MySQL provider available. Please direct any questions about this provider, including the release timeline, to the `MySQL Project <http://forums.mysql.com/list.php?38>`_.
+We are working with the MySQL team to make a MySQL provider available. 
 
+The official MySQL provider announcement with initial support for .NET Core and EF Core is available at http://insidemysql.com/mysql-connectornet-7-0-4-m3-has-been-released/
+
+Please direct any questions about this provider, including the release timeline, to the `MySQL Project <http://forums.mysql.com/list.php?38>`_.
```
Comment (ErikEJ) [inline docs/providers/mysql/index.rst]: No need to mention releasetimeline?

File: docs/providers/mysql/index.rst [hunk 2]
```diff
@@ -1,6 +1,17 @@
 MySQL (Coming Soon)
 ===================
 
-We are working with the MySQL team to make a MySQL provider available. Please direct any questions about this provider, including the release timeline, to the `MySQL Project <http://forums.mysql.com/list.php?38>`_.
```
Comment (ErikEJ) [inline docs/providers/mysql/index.rst]: Remove coming soon from the heading

File: docs/providers/mysql/index.rst [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+Please direct any questions about this provider, including the release timeline, to the `MySQL Project <http://forums.mysql.com/list.php?38>`_.
+
+Commercial MySQL Provider(s)
+----------------------------
 DevArt dotConnect does offer a paid provider for MySQL, see :doc:`/providers/devart/index` for details.
+
+
+Other Open Source MySQL Provider(s)
+-----------------------------------
+*  *Pomelo Foundation*: There is an open sourced and free mysql provider for ef core from *Pomelo Foundation* at https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql
```
Comment (ErikEJ) [inline docs/providers/mysql/index.rst]: Casing! There is also: https://www.nuget.org/packages/SapientGuardian.EntityFrameworkCore.MySql/

Comment (adebisi-fa) [inline docs/providers/mysql/index.rst]: Thanks. Fixed.

Comment (adebisi-fa) [inline docs/providers/mysql/index.rst]: Thanks! Fixed.

File: docs/providers/mysql/index.rst (hunk 2 above)
Comment (adebisi-fa) [inline docs/providers/mysql/index.rst]: Done, thanks!

File: docs/providers/mysql/index.rst (hunk 1 above)
Comment (adebisi-fa) [inline docs/providers/mysql/index.rst]: Noted. Thanks.

**Comment by rowanmiller** [comment]
Thanks for taking the time to put this together. The structure we have at the moment is one page per provider. This MySQL page is for the official MySQL provider, which I have https://github.com/aspnet/EntityFramework.Docs/pull/259 to update. I'm working with Pomelo folks to get their PR tidied up to add info on the Pomelo provider (https://github.com/aspnet/EntityFramework.Docs/pull/221). So closing this one out.

**Pull request closed without merging**